drive_to_gps/include/calculate_degrees.h

```diff
@@ -0,0 +1,38 @@
+#ifndef CALCULATE_DEGREES_H
```

Comment:
modern C++ dictates that u should use `#pragma once` instead of header guards. its pretty much universally supported by compilers.